lib/screens_web/controllers/screen_controller.ex

```diff
@@ -1,107 +1,7 @@
 defmodule ScreensWeb.ScreenController do
   use ScreensWeb, :controller
-  require Logger
-
-  alias Screens.Config.Cache
-  alias ScreensConfig.Screen
-
-  @default_app_id :solari
-  @app_ids ~w[solari]a
-  @app_id_strings Enum.map(@app_ids, &Atom.to_string/1)
-
-  plug(:body_class)
-  plug(:check_config)
-  plug(:environment_name)
-  plug(:last_refresh)
-
-  defp check_config(conn, _) do
-    if Cache.ok?() do
-      conn
-    else
-      conn
-      |> render_not_found()
-      |> halt()
-    end
-  end
-
-  defp last_refresh(conn, _, now \\ DateTime.utc_now()) do
-    timestamp = DateTime.to_iso8601(now)
-    assign(conn, :last_refresh, timestamp)
-  end
-
-  defp environment_name(conn, _) do
-    environment_name = Application.get_env(:screens, :environment_name)
-    assign(conn, :environment_name, environment_name)
-  end
-
-  defp body_class(conn, _) do
-    body_class =
-      case Map.get(conn.params, "scroll") do
-        "true" -> "scroll-enabled"
-        _ -> "scroll-disabled"
-      end
-
-    assign(conn, :body_class, body_class)
-  end
-
-  defp id_sort_fn(a, b) do
-    case {Integer.parse(a), Integer.parse(b)} do
-      {{m, ""}, {n, ""}} ->
-        m <= n
-
-      _ ->
-        a <= b
-    end
-  end
-
-  defp screen_ids(target_app_id) do
-    ids = Cache.screen_ids(&match?({_screen_id, %Screen{app_id: ^target_app_id}}, &1))
-
-    Enum.sort(ids, &id_sort_fn/2)
-  end
-
-  def index(conn, %{"id" => app_id})
-      when app_id in @app_id_strings do
-    app_id = String.to_existing_atom(app_id)
-
-    conn
-    |> assign(:app_id, app_id)
-    |> assign(:screen_ids, screen_ids(app_id))
-    |> render("index_multi.html")
-  end
-
-  def index(conn, %{"id" => screen_id} = params) do
-    is_screen = ScreensWeb.UserAgent.screen_conn?(conn, screen_id)
-
-    _ = Screens.LogScreenData.log_page_load(screen_id, is_screen)
-
-    case Cache.screen(screen_id) do
-      %Screen{app_id: app_id} ->
-        conn
-        |> assign(:app_id, app_id)
-        |> assign(:sentry_dsn, if(params["disable_sentry"], do: nil, else: Sentry.get_dsn()))
-        |> assign(:is_real_screen, match?(%{"is_real_screen" => "true"}, params))
-        |> assign(:requestor, params["requestor"])
-        |> render("index.html")
-
-      nil ->
-        render_not_found(conn)
-    end
-  end
-
-  def index(conn, _params) do
-    render_not_found(conn)
-  end
 
   def show_image(conn, %{"filename" => filename}) do
```

Comment:
Do we know what uses this endpoint? Curious if we can just get rid of this file altogether.

Reply:
It is definitely being used somehow in the wild, based on logs (search for `GET /image/`), so I'm inclined not to mess with it for now.